app/javascript/crayons/MentionAutocompleteTextArea/MentionAutocompleteTextArea.jsx

```diff
@@ -226,16 +227,14 @@ export const MentionAutocompleteTextArea = forwardRef(
 
     const handleTextInputChange = ({ target: { value } }) => {
       setTextContent(value);
-      const isComboboxVisible = !comboboxRef.current.classList.contains(
-        'hidden',
-      );
+      const isComboboxVisible =
+        !comboboxRef.current.classList.contains('hidden');
       const currentActiveInput = isComboboxVisible
         ? comboboxRef.current
         : plainTextAreaRef.current;
 
-      const { isUserMention, indexOfMentionStart } = getMentionWordData(
-        currentActiveInput,
-      );
+      const { isUserMention, indexOfMentionStart } =
+        getMentionWordData(currentActiveInput);
```

Comment:
Might be worth double-checking your Prettier is using the config at `./prettierrc.json` 🤔  Not a big deal, just curious why it's popping up with these

Reply:
It's because prettier has `"printWidth": 80` so it breaks early so it doesn't have to break the function.

Reply:
I guess it must have got missed in a previous commit hook 🤷‍♀️

Reply:
We should be locking down the prettier version or anytime we update it in Dependabot, we'd need to run it on the codebase. See https://prettier.io/docs/en/install.html#summary.